lib/internal/Magento/Framework/View/Element/Template.php

```diff
@@ -19,7 +19,7 @@
  * custom view models in block arguments in layout handle file.
  *
  * Example:
- * <block name="my.block" class="Magento\Backend\Block\Template" template="My_Module::template.phtml" >
+ * <block name="my.block" class="Magento\Framework\View\Element\Template" template="My_Module::template.phtml" >
```

Comment:
Since we are talking about the storefront area here, it makes sense to adjust this example. Thanks.